### PR TITLE
Switch to use new bundle format in olm manifest

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

**Comment by k-wall** [comment]
Aren't the image labels required?

https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md#bundle-dockerfile

File: olm-manifest/src/assembly/unix-dist.xml [hunk 1]
```diff
@@ -28,27 +28,23 @@
...
-          <directory>${project.basedir}/target/classes/manifests</directory>
-          <outputDirectory>/manifests</outputDirectory>
-        </fileSet>
     </fileSets>
     <files>
         <file>
-          <source>${project.basedir}/target/classes/latest/enmasse.package.yaml</source>
-          <outputDirectory>/manifests</outputDirectory>
-          <destName>${application.bundle.prefix}.package.yaml</destName>
+          <source>${project.basedir}/target/classes/annotations.yaml</source>
```
Comment (k-wall) [inline olm-manifest/src/assembly/unix-dist.xml:39]: What generates `annotations.yaml`?

Comment (lulf) [reply olm-manifest/src/assembly/unix-dist.xml:39]: I forgot to commit that :)

**Comment by lulf** [comment]
Release-0.33 branch was renamed to master: Please re-create PR against master branch.

**Pull request closed without merging**